app/api/emails.py

```
@@ -0,0 +1,141 @@
+        
+        logger.info(f"Email {email_record.id} submitted successfully")
+        
+        return EmailResponse(
+            email_id=email_record.id,
+            status=EmailStatus.QUEUED,
+            message="Email queued for sending",
+            created_at=email_record.created_at
+        )
+        
+    except ValueError as e:
+        logger.exception(f"Validation error: {str(e)}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)